src/miRBench/encoder.py

```diff
@@ -234,8 +234,8 @@ def prepare_dataset(self, df, miRNA_col, gene_col, batch_size=64, shuffle=False,
 
         for _, row in df.iterrows():
 
-            mirna_seq = reverse(row[miRNA_col].upper().replace("T", "U"))
-            mrna_seq = row[gene_col].upper().replace("T", "U")[0:40]
+            mirna_seq = row[miRNA_col].upper().replace("T", "U")
+            mrna_seq = reverse(row[gene_col].upper().replace("T", "U")[0:40])
```

Comment:
Are you defining the reverse function somewhere else Kat?

Reply:
Yes, it is here: https://github.com/katarinagresova/miRBench/blob/d68442ccbd82ebfe7bc297b989af5dda025d49bd/src/miRBench/encoder.py#L376